migrate/20240218_add_concession.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```

Comment:
Migrations need to be on their own commit so that we can deploy them separately. If we deploy the model without completing the migrations, app would crash, causing downtime until the migrations are applied.